specutils/tests/test_fitting.py

```diff
@@ -51,6 +89,27 @@ def test_single_peak_fit():
     x_single, y_single = single_peak()
     s_single = Spectrum1D(flux=y_single*u.Jy, spectral_axis=x_single*u.um)
 
+    # Fit the spectrum
+    g_init = estimate_parameters(s_single, models.Gaussian1D())
+
+    assert np.isclose(g_init.amplitude.value, 3.354169257846847)
+    assert np.isclose(g_init.mean.value, 6.218588636687762)
+    assert np.isclose(g_init.stddev.value, 1.608040201005025)
+
+    assert g_init.amplitude.unit == u.Jy
+    assert g_init.mean.unit == u.um
+    assert g_init.stddev.unit == u.um
+
```

Comment:
probably want to test the "non-supported model" case too, right?

Reply:
added one for MexicanHat1D and added more tests for the other "supported models"